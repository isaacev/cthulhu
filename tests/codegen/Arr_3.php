<?php

namespace Arr_3 {
  function main() {
    [
      "a",
      "b",
      "c"
    ];
  }
}

namespace {
  \Arr_3\main(null);
}
