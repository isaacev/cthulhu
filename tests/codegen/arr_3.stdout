<?php

namespace Kernel\Types {
  abstract class Maybe {}

  class Just extends \Kernel\Types\Maybe {
    function __construct($a) {
      $this->{0} = $a;
    }
  }

  class None extends \Kernel\Types\Maybe {}
}

namespace arr_3 {
  // #[entry]
  function main() {
    [
      "a",
      "b",
      "c"
    ];
  }
}

namespace {
  \arr_3\main();
}
